vmm/src/config.rs

```diff
@@ -644,6 +644,7 @@ impl CpusConfig {
         // list as it will always be checked for.
         #[allow(unused_mut)]
         let mut features = CpuFeatures::default();
+        #[allow(clippy::never_loop)]
```

Comment:
Do you want to take the chance to improve such code and avoid using more clippy overrides?